### ISSUE TITLE
[SUGGESTION] Option to disable tools binary check

### ISSUE THREAD

**Issue by MechDR** [lordmulder/Simple-x264-Launcher#29]
I'm trying to make this tool work like a pipe in Wine on Linux. The idea is, everything is done Wine/Windows side (indexing, frame server, AviSynth scripts), EXCEPT the encoding process, I'd rather have that done natively (performance wise, nothing against the Windows x264/x265 binaries :P :D). I though I'd fool the application by making a symbolic link in the appropriate place (places), but it turns out, it checks if the binaries are present, and no only that, but if they actually run :D. Didn't expect that :).

I believe my logic is solid? I mean... AviSynth+/VapourSynth should work the same in Windows and Linux, right? Not to mention the x264/x265 binaries, they have the same command set in both OSes, so they should output the same data.

So, basically, in order to see if this can actually work (I really, REALLY hope it does... that'll save me a lot of work, except compiling x265, which is no big deal, stable releases are not that regular), I need the binary check to be disabled.

I'm also opened to other suggestions of fooling Simple x264/x265 Launcher into thinking it's running a native Windows binary... or any other approaches to solve the problem :).

**Comment by lordmulder** [comment]
No response. Closing.

**Comment by MechDR** [comment]
Sorry for replying so late.

See, the problem is, all the in between tools and their configuring... it's just too much work, I've already got all of that working in Windows, why bothering configuring it natively in Linux. The only thing I have to manually install (and the real CPU eater) is the encoder, so why do everything natively.

My point being, the only thing I need working natively in Linux is the encoder, everything else is optional.

Regarding the question about compiling from source with the exe checks commented out, no, I haven't tried that. Will, try that and report ;).
